pkg/odo/cli/component/list.go

```diff
@@ -84,18 +80,21 @@ func (lo *ListOptions) Complete(name string, cmd *cobra.Command, args []string)
 
 		if util.CheckKubeConfigExist() {
 			klog.V(4).Infof("New Context")
-			lo.Context = genericclioptions.NewContext(cmd)
+			lo.Context = genericclioptions.NewContext(cmd, false, true)
 			lo.hasDCSupport, err = lo.Client.IsDeploymentConfigSupported()
 			if err != nil {
 				return err
 			}
+			fmt.Println("GIRISH")
```

Comment:
This and the following lines should be removed.

Reply:
I am aware, I am debugging it

Reply:
removed